packages/atchops/include/atchops/constants.h

```diff
@@ -3,4 +3,23 @@
 
 #define ATCHOPS_RNG_PERSONALIZATION "@atchops12345"
 
-#endif
\ No newline at end of file
+#include "mbedtls/md.h"
+
+typedef enum atchops_md_type {
+  ATCHOPS_MD_NONE = 0,
+  ATCHOPS_MD_MD5,
+  ATCHOPS_MD_RIPEMD160,
+  ATCHOPS_MD_SHA1,
+  ATCHOPS_MD_SHA224,
+  ATCHOPS_MD_SHA256,
+  ATCHOPS_MD_SHA384,
+  ATCHOPS_MD_SHA512,
+  ATCHOPS_MD_SHA3_224,
+  ATCHOPS_MD_SHA3_256,
+  ATCHOPS_MD_SHA3_384,
+  ATCHOPS_MD_SHA3_512,
+} atchops_md_type;
```

Comment:
These are what mbedtls makes available, not sure we need all of them, but I put them anyways. We do have runtime validation for some of them already.

Reply:
i think we should only expose the ones we use in the entirety of the atProtocol

i fear someone will come along and think they can just use any algorithm of their choosing (something intern me would do).

Reply:
I think it is useful to make these tools available. I believe the only situation where an unsupported hash type should fail right now is in rsa verify, since that has some per-algorithm implementation details.